src/testutils/contractController.ts

```diff
@@ -55,4 +58,10 @@ export class ContractController {
         Logger.debug(`ContractController.query [end]: function = ${interaction.getFunction()}, return code = ${bundle.returnCode}`);
         return bundle;
     }
+
+    private getTransactionHash(transaction: Transaction): string {
```

Comment:
Generally speaking, `hash` should also have been returned by `await this.provider.sendTransaction(transaction);`.

Reply:
done